unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarAnimatorLegacy.cs

```diff
@@ -113,6 +116,12 @@ void UpdateInterface()
 
         //NOTE(Brian): Vertical speed
         float verticalVelocity = flattenedVelocity.y;
+
+        if (Mathf.Abs(verticalVelocity) > float.Epsilon)
```

Comment:
What happens if I climb some stairs? wouldnt my vertical velocity be positive and (in some cases) higher than `float.Epsilon`?

Reply:
This is an additional check based on velocity, the raycast is still there, so we are not introducing another case.
Anyways I've tested it by removing all the "isOwnPlayer" conditionals and the only issue I see is when we go down from stairs, the character moves too fast and we have some frames where we are not grounded and the free fall animation triggers, this won't be too noticeable because the interpolation tends to make everyone fly anyways.

Reply:
I've added some comments to clarify its purpose